src/PowerShellEditorServices/Services/Symbols/Vistors/AstOperations.cs

```diff
@@ -62,7 +62,7 @@ internal static class AstOperations
             ILogger logger,
             CancellationToken cancellationToken)
         {
-            if (!s_completionHandle.Wait(0))
+            if (!s_completionHandle.Wait(0, cancellationToken))
```

Comment:
This shouldn't need the cancellation passed through, since it's a `Wait(0)` -- the token should never get a chance to cancel

Reply:
Tell that to the compiler that complained.

Reply:
I suppose it can't hurt here, but I don't like the idea of misleading code -- the cancellation token definitely has no utility here

Reply:
We could pass `CancellationToken.None` instead (which says explicitly this won't be cancelled).

Reply:
Nah I think what you've got works fine